src/bytes/bytes.go

```diff
@@ -6,30 +6,220 @@
 // It is analogous to the facilities of the strings package.
 package bytes
 
-import (
-	"internal/bytealg"
-	"unicode"
-	"unicode/utf8"
+//+gobra
+
+// @ import (
+// @	. "verification/utils/definitions"
+// @	b "verification/utils/bitwise"
+// @	sl "verification/utils/slices"
+// @	seqs "verification/utils/seqs"
+// @	sets "verification/utils/sets"
+// @ )
+
+// the following is some bookkeeping since we have not (yet) specified
+// "internal/bytealg", "unicode", or "unicode/utf8"
+//
+// each function of the respective package is rewritten to instead
+// call one of the stubs below. Each function follows the convention
+// `package_path__FunctionName`, where `package_path` is the path of the
+// package, with `/` replaced by `_`.
+/* @
+
+// utf8.Codepoints(s) returns the sequence of (utf-8 encoded) codepoints in `s`
+ghost
+pure
+ensures 0 < len(s) ==> 0 < len(res)
+ensures 0 == len(s) ==> 0 == len(res)
+decreases
+func unicode_utf8__Codepoints(s []byte) (res seq[rune])
+
+
+@ */
+
+//gobra:rewrite 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
+//gobra:cont
+//gobra:end-old-code 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
+
+// @ preserves acc(sl.Bytes(b, 0, len(b)), R50)
+//
+// @ ensures res >= 0
+//
+// @ ensures len(indices) == res
+//
+// @ ensures forall i int :: {i in indices} i in indices ==> 0 <= i && i < len(b)
+//
+// @ ensures forall i int :: {i in indices} i in indices ==> ( sl.GetByte(b, 0, len(b), i) == c)
+//
+// @ ensures forall i int :: {i in indices} !(i in indices) ==> ( i < 0 || len(b) <= i || sl.GetByte(b, 0, len(b), i) != c)
+func internal_bytealg__Count(b []byte, c byte) (res int /* @, ghost indices set[int] @ */)
+
+const internal_bytealg__MaxBruteForce = 64
+
+func internal_bytealg__Cutover(n int) int
```

Comment:
we don't want to change the code in the file, or make the file not compilable. We can discuss how to re-use the approach from VerifiedSCION in our next meeting to address this.